connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoSinkOptions.scala

```diff
@@ -57,7 +61,8 @@ object SchemaAdjustmentMode extends Enumeration {
   val NoAdjustment, FailIfNotMatch, GenerateDynamicCsvMapping = Value
 }
 
-case class WriteOptions(tableCreateOptions: SinkTableCreationMode.SinkTableCreationMode = SinkTableCreationMode.FailIfNotExist,
+case class WriteOptions(pollingOnDriver:Boolean = true,
```

Comment:
Shouldn't we tune the default for prod instead of dev?

Reply:
We want first time users to understand what is happening and have this experience

Reply:
I personally think the defaults should be for prod, and the README instructs the need to temporarily adjust for dev.